idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/integration/IntegrationSyntax.kt

```diff
@@ -16,20 +27,96 @@ import org.jetbrains.kotlin.types.KotlinType
  * [IntegrationSyntax] servers as an interoperability algebra to register compiler extensions and kotlin extensions regarding the Ide.
  */
 interface IntegrationSyntax {
+
   fun MetaIde.syntheticResolver(
-    f: (Project) -> arrow.meta.phases.resolve.synthetics.SyntheticResolver?
+    f: (Project) -> SyntheticResolver?
   ): ExtensionPhase =
-    object : SyntheticResolver {
-      override fun syntheticResolver(project: Project): arrow.meta.phases.resolve.synthetics.SyntheticResolver? =
-        f(project)
+    object : arrow.meta.ide.phases.integration.SyntheticResolver {
+      override fun syntheticResolver(project: Project): SyntheticResolver? = f(project)
```

Comment:
```suggestion
      override fun syntheticResolver(project: Project): SyntheticResolver? =
         f(project)
```
Nit: does this format look good to you?

Reply:
¯\\_(ツ)_/¯ 

think, since the same operation is performed here in the entire file in the body of the method, you can not transfer